src/pages/Account/Account.styles.tsx

```diff
@@ -0,0 +1,264 @@
+import { ExternalLink } from 'react-feather'
+import { Flex, Link, Text } from 'rebass'
+import styled from 'styled-components'
+
+import { ButtonPurpleDim } from '../../components/Button'
+import Logo from '../../components/Logo'
+import { ListLayout } from '../../ui/ListLayout'
+
+export const Status = styled(Flex)<{ status: string }>`
+  text-transform: uppercase;
+  padding: 4px;
+  font-size: 9px;
+  border-radius: 6px;
+  font-weight: 700;
+  border-width: 1.5px;
+  ${({ status }) => {
+    switch (status) {
+      case 'COMPLETED':
+      case 'CONFIRMED':
+      case 'CLAIMED':
+        return `
+        border-color: #0e9f6e;
+        color: #0e9f6e;`
+      case 'REDEEM':
+      case 'PENDING':
+        return `
+        border-color: #F2994A;
+        color: #F2994A;`
+      case 'CANCELLED':
+        return `
+        border-color: #F02E51;
+        color: #F02E51;`
+      default:
+        return `
+        border-color: #a7a0e4;
+        color: #a7a0e4;`
+    }
+  }};
+  border-style: solid;
+  height: fit-content;
+  line-height: 1;
+`
+
+export const HeaderRow = styled(ListLayout)`
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    display: none;
+  `};
+`
+
+export const CustomLinkIcon = styled(ExternalLink)`
+  color: ${({ theme }) => theme.text5};
+`
+
+export const StyledLink = styled(Link)`
+  font-size: inherit;
+  display: flex;
+  color: ${({ theme }) => theme.text4};
+  align-items: center;
+  :hover,
+  :active,
+  :focus {
+    text-decoration: none;
+    color: ${({ theme }) => theme.text2};
+  }
+`
+
+export const GridCard = styled(Flex)<{ status?: string }>`
+  padding: 24px 22px;
+  line-height: 18px;
+  align-items: center;
+  color: #c0baf6;
+  border-bottom: 1px solid #3e4259;
+  font-size: 16px;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    padding: 16px 10px;
+    &:last-of-type {
+      border-radius: 12px;
+    }
+    flex-wrap: wrap;
+    gap: 12px 4px;
+    font-size: 14px;
+  `};
+  background: ${({ status }) => {
+    switch (status) {
+      case 'COMPLETED':
+      case 'CONFIRMED':
+      case 'CLAIMED':
+        return 'linear-gradient(256.45deg, rgba(15, 152, 106, 0.2) 6.32%, rgba(15, 152, 106, 0) 65.79%)'
```

Comment:
Can you round this values ?  I think difference between `65.79%` and `66%` will be not noticeable but code will be more readable